l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 OpenBIG (<http://www.openbig.org>)
+     Copyright 2019-2020 Onestein (<https://www.onestein.eu>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_l10n_de_tax_report_form" model="ir.ui.view">
+        <field name="model">l10n.de.tax.statement</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field
+                        name="state"
+                        widget="statusbar"
+                        statusbar_visible="draft,posted,final"
+                    />
+                    <button
+                        name="statement_update"
+                        string="Update"
+                        states="draft"
+                        type="object"
+                        icon="fa-repeat"
+                    />
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="post"
+                            string="Post"
+                            states="draft"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-arrow-right text-success"
+                        />
+                        <button
+                            name="reset"
+                            string="To Draft"
```

Comment:
You changed this from 'Reset to Draft' to 'To Draft'. The `Reset` terminology is something of a convention of old to indicate the exceptional character of the state change. It is still used in Odoo on the account.move form, the payment form, and also in the NL counterpart of this module: https://github.com/OCA/l10n-netherlands/blob/16.0/l10n_nl_tax_statement/views/l10n_nl_vat_statement_view.xml#L36. Is the terminology change something of a personal preference or driven by functional considerations?

Reply:
It looked too long. But I was not aware of the convention, so I'll change that back.

Reply:
Thanks! Please squash it into the Migration to 16.0 commit.